### PR TITLE
Enable Slack Alerts in ODFE 1.10.0

### PR DESCRIPTION
**Pull request by peterzhuamazon**
*Issue #, if available:*
https://github.com/opendistro-for-elasticsearch/opendistro-infra/issues/249

*Description of changes:*
This PR is to enable slack alerts for ODFE 1.10.0 Version.

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/215713678

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/check-plugin.yml [hunk 1]
```diff
@@ -26,12 +26,19 @@ jobs:
...
+#        if: ${{ always() }}
+#        uses: ros-tooling/action-amazon-chime@master
+#        with:
+#          message: ${{ env.CHIME_MESSAGE }}
+#          webhook: ${{ secrets.CHIME_ODFE_RELEASE }}
+
+      - name: Send Slack Message
         if: ${{ always() }}
-        uses: ros-tooling/action-amazon-chime@master
+        uses: peterzhuamazon/github-slack-action@master
```
Comment (weicongs-amazon) [inline .github/workflows/check-plugin.yml:38]: what's the long term plan for this action?

File: .github/workflows/check-plugin.yml [hunk 2]
```diff
@@ -26,12 +26,19 @@ jobs:
           echo ::set-env name=CHIME_MESSAGE::$(cat /tmp/chime_message.md)
           exit `cat /tmp/plugin_status.check`
 
-      - name: Send Chime Message
+#      - name: Send Chime Message
+#        if: ${{ always() }}
+#        uses: ros-tooling/action-amazon-chime@master
+#        with:
+#          message: ${{ env.CHIME_MESSAGE }}
+#          webhook: ${{ secrets.CHIME_ODFE_RELEASE }}
```
Comment (weicongs-amazon) [inline .github/workflows/check-plugin.yml:34]: remove these if they won't be used anymore

File: .github/workflows/check-plugin.yml (hunk 1 above)
Comment (peterzhuamazon) [reply .github/workflows/check-plugin.yml:38]: It is just like every single action we grab from the market. Leave it there.
I do not see an issue here.

File: .github/workflows/check-plugin.yml (hunk 2 above)
Comment (peterzhuamazon) [reply .github/workflows/check-plugin.yml:34]: We still have chime do we really want to completely remove it?

Comment (weicongs-amazon) [reply .github/workflows/check-plugin.yml:34]: I don't see the need to keep it. Let's remove it.

File: .github/workflows/check-plugin.yml (hunk 1 above)
Comment (weicongs-amazon) [reply .github/workflows/check-plugin.yml:38]: I see. not sure how this action is managed. I think it might be better to mange it at org level. It's ok for me to keep it in personal repo for now.

Comment (gaiksaya) [reply .github/workflows/check-plugin.yml:38]: Can it be retrieved from a folder of a repo? If yes we can add it to build repo as a folder

Comment (peterzhuamazon) [reply .github/workflows/check-plugin.yml:38]: @gaiksaya It is a fork from a community member's existing action.
I think it is better to store it in my public repo and getting the upstream updates if possible.

File: .github/workflows/check-plugin.yml (hunk 2 above)
Comment (peterzhuamazon) [reply .github/workflows/check-plugin.yml]: Resolved.

File: .github/workflows/check-plugin.yml (hunk 1 above)
Comment (camerski) [reply .github/workflows/check-plugin.yml:31]: What will happen if you go on vacation or leave the company? Will we be able to maintain this?

File: bin/version.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+    "current":"7.9.0",
+    "next": "7.9.1"
   }, 
   "odVersion": {
-    "previous": "1.8.0",
-    "current": "1.9.0",
-    "next": "1.9.1"
+    "previous": "1.9.0",
+    "current": "1.10.0",
+    "next": "1.10.1"
```
Comment (camerski) [inline bin/version.json:10]: These changes should be in a standalone PR. They are not part of the slack change.

File: .github/scripts/check_plugin.sh [hunk 4]
```diff
@@ -41,9 +41,9 @@ ODFE_VERSION=$2
 OLDIFS=$IFS
 
 # This script allows users to manually assign parameters
-if [ "$#" -eq 0 ] || [ "$#" -gt 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
+if [ "$#" -gt 2 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
 then
-  echo "Please assign at least 1 / at most 2 parameters when running this script"
+  echo "Please assign at most 2 parameters when running this script"
```
[the remark this answers is not in the document]
Comment (peterzhuamazon) [reply .github/scripts/check_plugin.sh:46]: It will fail the alerts if I dont fix this bug, introduced a while ago in a refactoring PR.

File: bin/version.json (hunk 3 above)
Comment (peterzhuamazon) [reply bin/version.json:10]: Until Next Time 👍
